applications/zigbee_weather_station/README.rst

```diff
@@ -134,34 +134,36 @@ CONFIG_WEATHER_CHECK_PERIOD_SECONDS - How often sensor data is read
 Zigbee weather station build types
 ==================================
 
-The Zigbee weather station application does not use a single :file:`prj.conf` file.
+The Matter weather station application does not use a single :file:`prj.conf` file.
```

Comment:
Is this change intentional? I'm not involved in Zigbee development for about 1.5 year, so I might be not aware of something but remaining occurrences of "Zigbee" are left unchanged.

Reply:
No, this is my bad, thanks for spotting!